packages/lib-classifier/src/components/Classifier/components/Banners/components/SubjectSetProgressBanner/SubjectSetProgressBanner.spec.js

```diff
@@ -19,26 +19,22 @@ describe('Component > SubjectSetProgressBanner', function () {
 
     it('should indicate your position within the subject set', function () {
       const component = render(<Default />)
-      const bannerText = 'Subject 37/56'
+      const bannerText = 'Hello there!: Subject 37/56'
       expect(component.getByText(bannerText)).to.exist()
     })
 
     describe('when the subject is already seen', function () {
       it('should show the Already Seen banner', function () {
-        // Unsure why RTL isn't picking up the updated args from the story
-        // so explicitly setting already_seen to true here
```

Comment:
The default story args were imported as `WithAlreadySeenSubject.args`, but weren't passed to the rendered component as props, so I've added that to the tests here.

Reply:
I don't understand this. There aren't any default story args. I removed them from the default export of the stories and defined them explicitly on each story. It seems like you're doing the exact same thing I did, but in a different way. I would've expected the static `args` defined in the stories would've been already defined and imported.

Reply:
I'm taking the static `WithAlreadySeenSubject.args`, imported with the story, and passing them to `WithAlreadySeenSubject` when it runs. They aren't explicitly defined as default props for the `WithAlreadySeenSubject` functional component, so they have to be explicitly passed in as props. Storybook does that for you, but React will use the defaults that are declared in the function definition.

If the story function was declared with default props, React would use those and you could render the component without having to set props:
```js
export function WithAlreadySeenSubject({ already_seen = true, dark = false, retired = false })
```
I'm not really sure why storybook doesn't use ES6 destructured defaults here. Maybe so it can support objects and complex types as default values?